api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/FhirPractitionerDaoImpl.java

```diff
@@ -35,7 +36,12 @@
 	@Inject
 	@Named("sessionFactory")
 	private SessionFactory sessionFactory;
-	
+
+	@Override
+	public List<Provider> findAllProviders() {
+		return providerService.getAllProviders();
```

Comment:
Could we use the pattern `sessionFactory.getCurrentSession().createCriteria(...).list();` here?